lib/client-assets.php

```diff
@@ -153,7 +153,8 @@ function wc_admin_print_script_settings() {
 	}
 
 	$preload_data_endpoints = array(
-		'countries' => '/wc/v3/data/countries',
+		'countries'              => '/wc/v3/data/countries',
```

Comment:
should these be `/v4` now?

Reply:
This got updated on the rebase, but actually breaks. I opened up https://github.com/woocommerce/wc-admin/pull/1354 to address it.